packages/components/MenuButton/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fluentui-react-native/menu-button",
+  "version": "0.0.0",
+  "description": "A cross-platform MenuButton component using the Fluent Design System",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/button": "^0.17.6",
+    "@fluentui-react-native/contextual-menu": "^0.9.0",
+    "@fluentui-react-native/icon": "^0.7.5",
+    "@uifabricshared/foundation-composable": ">=0.8.2 <1.0.0",
+    "@uifabricshared/foundation-compose": "^1.9.4",
+    "@uifabricshared/foundation-settings": ">=0.9.1 <1.0.0"
+  },
+  "devDependencies": {
+    "@types/react-native": "^0.63.0",
+    "@uifabricshared/build-native": "^0.1.1",
+    "@uifabricshared/eslint-config-rules": "^0.1.1",
+    "react-native": "^0.63.4"
```

Comment:
Seems like this is a dependency and not a dev dependency in this package

Reply:
I see that react-native is in devDependencies in all the components and also it's in peerDependencies. Should I move it to dependencies anyway?

Reply:
I didn't add it for now, because we have react-native in peerDependencies

Reply:
This is out of scope of this PR, but I personally feel we should have react-native a peer dependency of all our packages. It would make it easier in the future (And for clients) for our components to not accidentally pull in a conflicting version of react-native. But this should be fine since it's following what the rest of our controls are doing.